src/main/resources/db/migration/V9_08__delete_avlysning.sql

```diff
@@ -0,0 +1,15 @@
+DELETE
+FROM motedeltaker_arbeidstaker_varsel
+WHERE id = '221126';
+
+DELETE
+FROM motedeltaker_arbeidsgiver_varsel
+WHERE id = '221126';
+
```

Comment:
Det ligger en rad i pdf-tabellen for hver av disse radene (med id 678784 og 678785). Kan vurdere å slette dem også.